### PR TITLE
PHP agent release 10.2.0 (Maybe publish October 3rd or 4th?)

### PR DESCRIPTION
**Pull request by misiekn**
Updates for the PHP agent v10.2.0:

* Add 'PHP agent v10.2.0' release notes
* Update PHP agent EOL policy

### CODE REVIEW

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.2.0.314'
+---
+## New Relic PHP Agent v10.2.0.314
+
+### New Features
+* newrelic-daemon: Informational messages will be logged if the number of connected applications falls below 25 or exceeds 200. [#534](https://github.com/newrelic/newrelic-php-agent/pull/534)
+
+### Bug Fixes
+* agent: Fixed SIGSEGV in Drupal 7 drupal_http_request instrumentation for PHP 7.4, 8.0 and 8.1. [#538](https://github.com/newrelic/newrelic-php-agent/pull/538)
+* agent: Fixed displaying newrelic.application_logging*.enabled value in php info output. [#535](https://github.com/newrelic/newrelic-php-agent/pull/535)
```
Comment (akristen) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx:14]:
```suggestion
* agent: Fixed displaying newrelic.application_logging*.enabled value in PHP info output. [#535](https://github.com/newrelic/newrelic-php-agent/pull/535)
```

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+version: 10.2.0.314
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.2.0.314'
+---
+## New Relic PHP Agent v10.2.0.314
+
+### New Features
+* newrelic-daemon: Informational messages will be logged if the number of connected applications falls below 25 or exceeds 200. [#534](https://github.com/newrelic/newrelic-php-agent/pull/534)
+
+### Bug Fixes
+* agent: Fixed SIGSEGV in Drupal 7 drupal_http_request instrumentation for PHP 7.4, 8.0 and 8.1. [#538](https://github.com/newrelic/newrelic-php-agent/pull/538)
```
Comment (akristen) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx:13]:
```suggestion
* agent: Fixed SIGSEGV in Drupal 7 `drupal_http_request` instrumentation for PHP 7.4, 8.0 and 8.1. [#538](https://github.com/newrelic/newrelic-php-agent/pull/538)
```

File: src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx (hunk 1 above)
Comment (akristen) [inline src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx:14]:
```suggestion
* agent: Fixed displaying `newrelic.application_logging*.enabled` value in PHP info output. [#535](https://github.com/newrelic/newrelic-php-agent/pull/535)
```

Comment (akristen) [reply src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx:14]: Not sure if this should be in a codeblock

Comment (misiekn) [reply src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx]: @akristen Yes, it makes sense to be a code block. Thank you for applying this formatting.

Comment (rhetoric101) [reply src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx]: @akristen and @misiekn, I added the code formatting.

**Comment by rhetoric101** [comment]
@misiekn when would you like us to publish these PHP changes?

**Comment by misiekn** [comment]
Hi @rhetoric101! The New Relic PHP Agent 10.2.0 is scheduled to be released today - 10/3.

**Pull request merged**